src/routes/main.js

```diff
@@ -71,21 +70,6 @@ export default [
       default: GroupCreate,
     },
   },
-  {
-    name: 'historyDetail',
-    path: '/history/:historyId',
```

Comment:
By removing the route, you break existing URLs, which I consider bad user experience and generally not OK. It might be ok in this special case (I suppose that history detail URLs haven't been shared among people), but it would be better to have at least some kind of fallback (alias or redirect).